docs/installation-contributors.md

```diff
@@ -3,17 +3,18 @@ id: installation-contributors
 title: Installation
 ---
 
-> ⚠ ️ This project is very alpha stage. Expect bugs and incomplete documentation.
+> ⚠ ️ This project is very alpha stage. Expect bugs and incomplete
+> documentation.
 
 The following instructions are intended for contributors who want to try Metals
 and provide feedback. We do not provide support for day-to-day usage of Metals.
 
 ## Requirements
 
-* Scala 2.12.4, down to the exact PATCH version. Note that Scala versions 2.11.x and
-  2.12.6 are not supported.
-* Java 8. Note that Java 9 or higher has not been tested.
-* macOS or Linux. Note that there have been reported issues on Windows.
+- Scala 2.12.7, down to the exact PATCH version. Note that Scala versions 2.11.x
```

Comment:
maybe make the Scala version dynamic using `V.scala212`?

Reply:
Good idea, done.

Reply:
The docs don't build actually at the moment because the `Configuration` object is now gone. We'll need to update the docs before there is a release.